s2spy/utils.py

```diff
@@ -169,3 +169,54 @@ def get_month_names() -> Dict:
         "nov": 11,
         "dec": 12,
     }
+
+
+def check_month_day(month: int, day: int = 1):
+    """Checks if the input day/month combination is valid.
+
+    Months must be between 1 and 12, and days must be within 1 and 28/30/31 (depending
+    on the month).
+
+    Args:
+        month: Month number
+        day: Day number. Defaults to 1.
+    """
+
+    if month in {1, 3, 5, 7, 8, 10, 12}:
+        if (day < 1) or (day > 31):
+            raise ValueError(
+                "Incorrect anchor input. "
+                f"Day number {day} is not a valid day for month {month}"
+            )
+    elif month in {4, 6, 9}:
+        if (day < 1) or (day > 30):
+            raise ValueError(
+                "Incorrect anchor input. "
+                f"Day number {day} is not a valid day for month {month}"
+            )
+    elif month == 2:
+        if (day < 1) or (day > 28):
```

Comment:
(Clearly you've thought about it already) Feb 29th is really tricky for the setup...It is ok to put it this way, to simply block the user from doing tricky things 😂.

Reply:
Yes I don't think we want to consider leap days as valid input!